<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ASCINLAP61473" failures="0" tests="5" name="com.fry.test.SnapdealEndToEndTest" time="0.017" errors="1" timestamp="2025-10-24T12:35:33 IST" skipped="4">
  <testcase classname="com.fry.test.SnapdealEndToEndTest" name="setup" time="0.016">
    <error message="Driver must be set" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Driver must be set
at org.openqa.selenium.internal.Require.nonNull(Require.java:63)
at org.openqa.selenium.interactions.Actions.<init>(Actions.java:67)
at org.openqa.selenium.interactions.Actions.<init>(Actions.java:63)
at com.fry.test.SnapdealEndToEndTest.setup(SnapdealEndToEndTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="com.fry.test.SnapdealEndToEndTest" name="loginWithMobileNumber" time="0.000">
    <skipped/>
  </testcase> <!-- loginWithMobileNumber -->
  <testcase classname="com.fry.test.SnapdealEndToEndTest" name="addBluetoothSpeakerToCart" time="0.000">
    <skipped/>
  </testcase> <!-- addBluetoothSpeakerToCart -->
  <testcase classname="com.fry.test.SnapdealEndToEndTest" name="addSportsShoesToCart" time="0.001">
    <skipped/>
  </testcase> <!-- addSportsShoesToCart -->
  <testcase classname="com.fry.test.SnapdealEndToEndTest" name="verifyCartAndLogout" time="0.000">
    <skipped/>
  </testcase> <!-- verifyCartAndLogout -->
</testsuite> <!-- com.fry.test.SnapdealEndToEndTest -->
